### DISCUSSION TITLE
"View all screens" background color

### DISCUSSION THREAD

**Discussion by cargeo** [MaxBazarov/demo-station-plugin#5] [General]
Since recently the page color is used as background in "View all screens" mode. It was transparent before. Is there a way to make it transparent again. Maybe as an option in settings?

**Comment by MaxBazarov** [comment]
Ok, thanks for the comment.
I released the new version just now with the new setting. Uncheck it to get dark background back. 
<img width="401" alt="Set" src="https://github.com/MaxBazarov/demo-station-plugin/assets/18418306/efa52049-d44e-41a4-bdb6-2db7f1cd7769">

**Reply by cargeo** [reply]
Thank you. 
Noticed: Gallery title is now "Test"

**Reply by MaxBazarov** [reply]
> Noticed: Gallery title is now "Test"
Fixed.
Also I want to mention that mouse wheel zooming works now in Gallery. It doesn't work well on large projects for now, but it's better than nothing.
